services/QuillLMS/spec/support/shared/quill_big_query/test_runner.rb

```diff
@@ -54,7 +54,7 @@ def execute(query)
       attr_type = record.class.column_for_attribute(attr).type
 
       if value.nil?
-        "''"
+        "NULL"
```

Comment:
At one [point](https://github.com/empirical-org/Empirical-Core/pull/10605/commits/53e6e1208538696274449cf0f01db2111c23c84c#diff-33db411271006d2a00f8922a18eee3429ddf7a8e545c472b0f36c835faf2a4e0R43) in building out the the test runner, I had `'NULL'` in here like this but couldn't figure out why it wasn't working properly (probably the disjunction with IS NULL).  Nice work on figuring it out.